scripts/kconfig/kconfigfunctions.py

```diff
@@ -765,6 +765,24 @@ def dt_compat_any_has_prop(kconf, _, compat, prop):
 
     return "n"
 
+def dt_compat_any_has_prop_val(kconf, _, compat, prop, value):
```

Comment:
why a new function for this ?
To me this looks like a fine case where different number of arguments can be used, see other comments.

Reply:
thanks for pointing it out